templates/csharp/message-extension-copilot/GettingStarted.md

```diff
@@ -23,6 +23,7 @@
    2. When Teams launches in the browser, click the Apps icon from Teams client left rail to open Teams app store and search for Copilot.
    3. Open the Copilot app and send a prompt to trigger your plugin.
    4. Send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
+      > Note: This message may not trigger the message extension in Copilot depending on various factors, if this happens, you can try another message or send feedback to us by clicking the thumbs down button in the response card. We appreciate your opinions and suggestions. Thank you!
 
```

Comment:
Note: This prompt may not always make Copilot include a response from your message extension. If it happens, try some other prompts or leave a feedback to us by thumbing down thee Copilot response and leave a message tagged with `[MessageExtension]`.

Reply:
Updated.